docs/product/explore/session-replay/web/replay-page-and-filters.mdx

```diff
@@ -29,3 +29,19 @@ Each replay will bring users to the [**Replay Details**](/product/explore/sessio
 All replays have built-in key fields, and may have [custom tags](/platforms/javascript/enriching-events/tags/) if you have them set up. The search bar operates similarly to the one in the Discover [Query Builder](/product/explore/discover-queries/query-builder/#filter-by-search-conditions). Use it to enter these keys and assign them values. This will filter down your list of replays so you can quickly find the ones with the criteria you care about. For example, you can refine your replays search to a specific browser by using `browser.version:` as a key field and assigning the version as a value. For a list of all the built-in key fields/replay properties, check out our [Searchable Properties](/concepts/search/searchable-properties/#replay-properties) documentation.
 
 ![Highlight the Search Filters input on the Session Replay Index Page](./img/replay-index-filtering.png)
+
+## Deleting Replays
+
+Replays can be deleted either individually or in bulk. Deleting replays in bulk requires project:write or project:admin access.
+
+When a replay is deleted, it is permanently removed from Sentry's servers, with the exception of its metadata. The metadata rows are archived and remain inaccessible to users. They are retained only until they are removed according to your organization's data retention policy. These metadata rows may also be eagerly deleted, but doing so requires manual intervention. To request this, please file a support ticket.
+
+To delete a single replay, go to the replay details page and click the "Delete" link, which is located under the "..." (more options) button.
+
+To delete multiple replays at once, go to the index page and use the checkboxes next to each replay row to select the ones you want to delete. Then click the "Delete" button to remove the selected replays. A pop up will appear which will direct you to a dedicated UI for monitoring the bulk delete job's progress.
```

Comment:
I think we should use of the copy from the PR description here as well! The permissions info is relevant and so is where the Settings page is 

<img width="698" height="134" alt="Screenshot 2025-08-25 at 1 28 12 PM" src="https://github.com/user-attachments/assets/bbdd30ef-aadc-4813-b43e-fd925d0efd33" />